hal/spi/spi_drv_stm32.h

```diff
@@ -72,14 +72,15 @@
 #define APB2_CLOCK_ER     (*(volatile uint32_t *)(0x40021034))
 #define APB2_CLOCK_RST    (*(volatile uint32_t *)(0x40021024))
 #define RCC_GPIO_CLOCK_ER (*(volatile uint32_t *)(0x4002102C))
+#define GPIOA_BASE (0x50000000)
```

Comment:
Any concern about this change breaking any existing users?

Reply:
You are right. I've reverted the original values and left an instance with the values I've used to test locally under ifdef.